pkgs/development/libraries/gmock/default.nix

```diff
@@ -1,15 +1,13 @@
-{ stdenv, fetchurl, unzip, cmake}:
-
+{ stdenv, cmake, callPackage }:
+let
+  source = callPackage ./source.nix { };
```

Comment:
The thing is that `fetchurl` only fetches, but `fetchzip` (and derived functions) also unpacks the archive afterwards, all in a single step. So, `gtest` could also use `fetchzip` instead of the whole gtest-src derivation. It might be just because `fetchzip` is relatively new and anyway, its main purpose is to fight non-deterministic archives.

See https://github.com/eduarrrd/nixpkgs/pull/1